python/scripts/cirkopt.py

```diff
@@ -64,6 +65,13 @@ def _add_common_args(parser: argparse.ArgumentParser):
         dest="loglevel",
         const=INFO,
     )
+    parser.add_argument(
+        "--outindex",
```

Comment:
nit: i think outindex doesn't mean much, would be nice if it was called delay index or something like that

Reply:
but it isn't just delay, it is basically an index that determines the test bench parameters, where the first index chooses the input slope, and the second index chooses the capacitance. In liberate they are labelled "index_1" and "index_2". I agree the naming is bad, but nothing better comes to mind.